New episode for tidy data

jt14den opened the pull request.

jt14den committed:
tidy-data-episode

chennesy reviewed (changes requested):
Thanks @jt14den, the content here is looking good!

Could you try to render this locally first though (by adding the episode to your config.yaml), and see if you can troubleshoot some workbench errors? When I try to preview the episode I'm getting a bunch of purrr::map() errors that seem to be related to the HTML tables you've included to show the output of the DataFrames. It seems that some of the errors are related to the use of `<th>` in non-header rows where there should be `<td>` cells, but I couldn't eliminate all of the errors. (The rowspan attribute may also be causing problems). 

Anyway, you might need to include the tables in Markdown instead of HTML, or see if you can radically simplify the HTML. I couldn't find examples of Pandas Carpentries lessons showing DataFrame output in tabular format, which might be due to some of the issues we're having.

I'll have some copy-edits once that's up and running, but I think we just need to figure out this technical bit first...

thanks again!

chennesy committed:
Update tidy.md

restructuring HTML tables to avoid workbench errors

chennesy committed:
add config and image

chennesy committed:
merge tidy.md

chennesy commented:
@jt14den Assigning this back to you! I was able to make a few commits to address the workbench errors that the tables were presenting, and added the episode to the config file. Could you test this all on your end before we move it out of draft phase and approve?

chennesy committed:
Update tidy.md

Converted tables to markdown. Used pandas .to_markdown() to capture these, though it's not perfect. Required some editing.
